solr/solrj/src/java/org/apache/solr/client/solrj/impl/Http2SolrClient.java

```diff
@@ -722,12 +717,12 @@ private boolean wantStream(final ResponseParser processor) {
       int httpStatus = response.getStatus();
 
       switch (httpStatus) {
-        case HttpStatus.SC_OK:
-        case HttpStatus.SC_BAD_REQUEST:
-        case HttpStatus.SC_CONFLICT: // 409
+        case HttpStatus.OK_200:
+        case HttpStatus.BAD_REQUEST_400:
+        case HttpStatus.CONFLICT_409: // 409
```

Comment:
```suggestion
        case HttpStatus.CONFLICT_409:
```

nit: don't need the comment `// 409` anymore its in the constant now.